mgmt/rest/v1/rbody/body.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/urfave/negroni"
 )
 
+const deprecationInfo = "The client API V1 has been depricated. Find more information here: https://github.com/intelsdi-x/snap/issues/1637"
```

Comment:
Okay, I updated the message, but I did not provide a date... The deprecation is pending on snap-client-go to be made public and API v2 tribe support, so I am just leaving the link to the issue that provides user details and the message can be updated when a specific date is actually known.